src/layout/Group/GroupSummary.tsx

```diff
@@ -90,16 +90,18 @@ export const GroupSummary = ({ componentNode, hierarchyLevel = 0, summaryOverrid
       className={cn(classes.groupContainer, { [classes.nested]: isNestedGroup })}
       data-testid={dataTestId}
     >
-      <Heading
-        size={isNestedGroup ? 'xsmall' : 'small'}
-        level={headingLevel}
-      >
-        <Lang
-          id={summaryTitle ?? title}
-          node={componentNode}
-        />
-      </Heading>
-      <Grid
+      {(summaryTitle || title) && (
```

Comment:
Nice! :raised_hands: I just found the same problem when working on the `Option` component. :partying_face: